grpc-sys/build.rs

```diff
@@ -129,10 +127,9 @@ fn build_grpc(cc: &mut Build, library: &str) {
         config.define("gRPC_BENCHMARK_PROVIDER", "none");
         if cfg!(feature = "openssl") {
             config.define("gRPC_SSL_PROVIDER", "package");
-            config.define("EMBED_OPENSSL", "false");
```

Comment:
Why?

Reply:
EMBED_OPENSSL is not used

Reply:
We can check it in cmake log. It's not used in current grpc cmakelists